site/zenodo_rdm/previewer/mirador.py

```diff
@@ -19,15 +22,32 @@ def can_preview(file):
     # supported_extensions list needs . prefixed -
     preview_extensions = current_app.config["MIRADOR_PREVIEW_EXTENSIONS"]
     supported_extensions = ["." + ext for ext in preview_extensions]
-    return (
-        file.has_extensions(*supported_extensions) and file.record.data["is_published"]
-    )
+
+    return file.has_extensions(*supported_extensions)
 
 
 def preview(file):
     """Render template."""
+    media_file_name = file.filename + ".ptif"
+
+    ext = splitext(file.filename)[1].lower()[1:]
+    format = (
+        ext
+        if ext in current_app.config["IIIF_SIMPLE_PREVIEWER_NATIVE_EXTENSIONS"]
+        else "jpg"
+    )
+    size = LocalProxy(lambda: current_app.config["IIIF_SIMPLE_PREVIEWER_SIZE"])
+    url = f"{file.data['links']['iiif_base']}/full/{size}/0/default.{format}"
```

Comment:
```suggestion
    iiif_simple_url = f"{file.data['links']['iiif_base']}/full/{size}/0/default.{format}"
```

nit: so that we know that it's for the "fallback" simple IIIF use